bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoDeviceHandler.java

```diff
@@ -154,21 +164,23 @@ protected void updateChannels() {
     }
 
     @Override
-    protected State getNAThingProperty(@NonNull String channelId) {
+    protected State getNAThingProperty(String channelId) {
         try {
+            @Nullable
+            DEVICE theDevice = device;
```

Comment:
On one line : 
```
@Nullable DEVICE theDevice = device
```

Reply:
Sorry, I can't change that because Eclipse automatically cuts on 2 lines when I save the file.

Reply:
Maybe because of the generic